docs/tutorials/getting_started/tournament_results.rst

```diff
@@ -20,6 +20,10 @@ This tutorial will show you how to access the various results of a tournament:
 - Payoff difference means: the mean score differences.
 - Cooperation counts: the number of times each player cooperated.
 - Normalised cooperation: cooperation count per turn.
+- Normalised cooperation: cooperation count per turn.
+- State distribution: the count of each type of state of a match
```

Comment:
I've added that in the wording below. Repeated it for both which might be a bit overkill, happy to tweak further :)